Testing Support/FormattedExample.m

```diff
@@ -97,8 +97,8 @@ @interface Foo
     Update(UpdateType t, const Slice &v) : type(t), value(v.data(), v.size()) {}
 };
 
-/* Same deal here, don't mess with adding newlines after this comment block 
- * with a different comment style 
```

Comment:
these unfortunately produce a large amount of changes in our repo, but I can't seem to find which setting controls this behavior. I looked at all of the settings [here](https://clang.llvm.org/docs/ClangFormatStyleOptions.html) marked with `clang-format {XYZ}` where `XYZ` was 13-19